src/couch_mrview/src/couch_mrview.erl

```diff
@@ -279,20 +279,20 @@ query_all_docs(Db, Args) ->
     query_all_docs(Db, Args, fun default_cb/2, []).
 
 query_all_docs(Db, Args, Callback, Acc) when is_list(Args) ->
-    query_all_docs(Db, to_mrargs(Args), Callback, Acc);
+    Args1 = couch_mrview_util:validate_all_docs_args(Db, to_mrargs(Args)),
+    query_all_docs(Db, Args1, Callback, Acc);
 query_all_docs(Db, Args0, Callback, Acc) ->
     Sig = couch_util:with_db(Db, fun(WDb) ->
         {ok, Info} = couch_db:get_db_info(WDb),
         couch_index_util:hexsig(couch_hash:md5_hash(?term_to_bin(Info)))
     end),
     Args1 = Args0#mrargs{view_type = map},
-    Args2 = couch_mrview_util:validate_all_docs_args(Db, Args1),
```

Comment:
Remove validation here and validate once in fabric or in couch*http modules instead